code/modules/holiday/halloween.dm

```diff
@@ -1,276 +0,0 @@
-		playsound(loc, 'sound/spookoween/bats.ogg', 300, TRUE)
-		var/number = rand(1,3)
-		for(var/i=0,i < number,i++)
-			new /mob/living/simple_animal/hostile/retaliate/bat(loc)
-		trapped = 0
-
-	else if(trapped == ANGRY_FAITHLESS)
-		visible_message(span_userdanger("The closet bursts open!"))
-		visible_message(span_userdanger("<font size='5'>THIS BEING RADIATES PURE EVIL! YOU BETTER RUN!!!</font>"))
-		playsound(loc, 'sound/hallucinations/wail.ogg', 300, TRUE)
-		var/mob/living/simple_animal/hostile/faithless/F = new(loc)
-		trapped = 0
-		QDEL_IN(F, 120)
-
-	else if(trapped == INSANE_CLOWN)
-		visible_message(span_userdanger("<font size='5'>...</font>"))
-		playsound(loc, 'sound/spookoween/scary_clown_appear.ogg', 300, TRUE)
-		spawn_atom_to_turf(/mob/living/simple_animal/hostile/clown_insane, loc, 1, FALSE)
-		trapped = 0
-
-//don't spawn in crates
-/obj/structure/closet/crate/trigger_spooky_trap()
-	return
-
-/obj/structure/closet/crate/set_spooky_trap()
-	return
-
-
-////////////////////
-//Spookoween Ghost//
-////////////////////
-
-/mob/living/simple_animal/shade/howling_ghost
-	name = "ghost"
-	real_name = "ghost"
-	icon = 'icons/mob/mob.dmi'
-	maxHealth = 1e6
-	health = 1e6
-	speak_emote = list("howls")
-	emote_hear = list("wails","screeches")
-	density = FALSE
-	anchored = TRUE
-	incorporeal_move = 1
-	layer = 4
-	var/timer = 0
-
-/mob/living/simple_animal/shade/howling_ghost/Initialize(mapload)
-	. = ..()
-	icon_state = pick("ghost","ghostian","ghostian2","ghostking","ghost1","ghost2")
-	icon_living = icon_state
-	status_flags |= GODMODE
-	timer = rand(1,15)
-
-/mob/living/simple_animal/shade/howling_ghost/Life()
-	..()
-	timer--
-	if(prob(20))
-		roam()
-	if(timer == 0)
-		spooky_ghosty()
-		timer = rand(1,15)
-
-/mob/living/simple_animal/shade/howling_ghost/proc/EtherealMove(direction)
-	forceMove(get_step(src, direction))
-	setDir(direction)
-
-/mob/living/simple_animal/shade/howling_ghost/proc/roam()
-	if(prob(80))
-		var/direction = pick(NORTH,SOUTH,EAST,WEST,NORTHEAST,NORTHWEST,SOUTHEAST,SOUTHWEST)
-		EtherealMove(direction)
-
-/mob/living/simple_animal/shade/howling_ghost/proc/spooky_ghosty()
-	if(prob(20)) //haunt
-		playsound(loc, pick('sound/spookoween/ghosty_wind.ogg','sound/spookoween/ghost_whisper.ogg','sound/spookoween/chain_rattling.ogg'), 300, TRUE)
-	if(prob(10)) //flickers
-		var/obj/machinery/light/L = locate(/obj/machinery/light) in view(5, src)
-		if(L)
-			L.flicker()
-	if(prob(5)) //poltergeist
-		var/obj/item/I = locate(/obj/item) in view(3, src)
-		if(I)
-			var/direction = pick(NORTH,SOUTH,EAST,WEST,NORTHEAST,NORTHWEST,SOUTHEAST,SOUTHWEST)
-			step(I,direction)
-		return
-
-/mob/living/simple_animal/shade/howling_ghost/adjustHealth(amount, updating_health = TRUE, forced = FALSE)
-	. = 0
-
-///////////////////////////
-//Spookoween Insane Clown//
-///////////////////////////
-
-///Insane clown mob. Basically a clown that haunts you.
-/mob/living/simple_animal/hostile/clown_insane
```

Comment:
nooo not the floor cluwne

Reply:
Eheh, you wish it's actually a floor cluwne.